spec/helpers/application_helper_spec.rb

```diff
@@ -364,4 +364,50 @@
       end
     end
   end
+
```

Comment:
The methods we're testing are in the comments helper, so let's make spec/helpers/comment_helper_spec.rb and put the tests there instead.